meta-mender-core/classes/mender-sdimg.bbclass

```diff
@@ -97,7 +101,7 @@ python() {
             d.setVar('IMAGE_TYPEDEP_sdimg_append', " " + fs)
 }
 
-IMAGE_DEPENDS_sdimg = "${IMAGE_DEPENDS_wic} dosfstools-native mtools-native"
+IMAGE_DEPENDS_sdimg = "${MENDER_IMAGE_EXTRA_DEPENDS} ${IMAGE_DEPENDS_wic} dosfstools-native mtools-native"
```

Comment:
I believe you can get the same effect without adding this, by adding this to `local.conf`:

```
MACHINE_ESSENTIAL_EXTRA_RDEPENDS_append = " bcm2835-bootfiles"
```